packages/grpc-native-core/tools/buildgen/gen_build_yaml.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python2.7
+
+# Copyright 2020 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+import re
+import os
+import sys
+import yaml
+
+node_versions = ["4", "5", "6", "7", "8", "9", "10", "11", "12", "13"]
+electron_versions = ["1.0", "1.1", "1.2", "1.3", "1.4", "1.5", "1.6", "1.7", "1.8", "2.0", "3.0", "3.1", "4.1", "4.2", "5.0", "6.0", "6.1", "7.0", "7.1"]
```

Comment:
Latest electron release is 8.0, so might as well?

Reply:
Done. And now that commit demonstrates what a change to add a version will look like.